cypress/integration/calculator.spec.js

```diff
@@ -0,0 +1,79 @@
+const calculate = (number1, operator, number2, expectedResult) => {
+  cy.get('.modifiers').click();
+  for (let index = 0 ; index < number1.length ; index += 1) {
+    cy.get('.digit').contains(number1[index]).click();
+  }
+	cy.get('.operations').contains(operator).click();
+  for (let index = 0 ; index < number2.length ; index += 1) {
+    cy.get('.digit').contains(number2[index]).click();
+  }
+	cy.get('.operations').contains('=').click();
+	cy.get('#total').should('have.text', expectedResult)
+}
+
+describe('계산기 테스트', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+	it('1. 숫자 입력 시 3자리까지만 결과창에 표시', () => {
+    cy.get('.digit').contains('1').click();
+    cy.get('#total').should('have.text', '1');
+    cy.get('.digit').contains('2').click();
+    cy.get('#total').should('have.text', '12');
+    cy.get('.digit').contains('3').click();
+    cy.get('#total').should('have.text', '123');
+    cy.get('.digit').contains('4').click();
+    cy.get('#total').should('have.text', '123');
+    cy.get('.digit').contains('5').click();
+    cy.get('#total').should('have.text', '123');
+  });
+
+  it('2. [AC] 입력 시 0으로 초기화', () => {
+    cy.get('.digit').contains('4').click();
+    cy.get('.digit').contains('2').click();
+    cy.get('.modifiers').click();
+    cy.get('#total').should('have.text', '0');
+  })
+
+  it('3. 연산자는 1개만 입력 가능', () => {
```

Comment:
```suggestion
  it('3. 연산자 2개 이상 입력시, 먼저 입력한 연산자로 적용', () => {
```
작성한 it의 설명은 함수 단위 테스트 설명이라고 생각합니다. BDD인 만큼 조금 더 세부적으로 행동을 중심으로 작성하면 좋을것 같아요! 아래는 Given, When, Then 에 대한 간략 설명입니댜!

Given : 시나리오 진행에 필요한 값을 설정합니다.
When : 시나리오를 진행하는데 필요한 조건을 명시합니다
Then : 시나리오를 완료했을 때 보장해야 하는 결과를 명시합니다.

Reply:
@hochan222

그러면 여기서는
  - Given: 연산자 2개(+, -),
  - When: 입력(클릭),
  - Then: 먼저 입력한 연산자 적용

이 되는 건가요?

Reply:
제가 코멘트를 단 이유는 Given, When, Then의 느낌보다는 다음 it은 유닛테스트 느낌이 더 강해서 였습니다. 조금 더 자세히 적어서 it 설명만 보아도 의미를 알 수 있어야 한다고 생각합니다. 

연산자 2개 이상(Given) 
입력시,(When)
먼저 입력한 연산자로 적용(Then)

으로 나누어서 적었습니다!